__tests__/example.test.ts

```diff
@@ -0,0 +1,28 @@
+'use-strict';
+import * as testlib from '../src/validate';
+import * as testreq from '../src/requests';
+
+const {smarthome} = require('actions-on-google');
+const app = smarthome();
+// Test passes
+test('Test 1: Sync Response passes', async () => {
```

Comment:
You don't need the prefix here or a number for the test. Just a label is sufficient.